File: php/1st_week/Praktikum/latihanpraktikum1b_php/DataTim.php
<?php

class DataTim{
    //properties class
    private $nama = ""; 
    private $negara = "";
    private $kota = ""; 
    private $kandang = ""; 
    private $tahunBerdiri = ""; 
    private $kaptenTim = ""; 
    private $pelatih = "";

    //constructor class
    public function __construct($nama = "", $negara = "", $kota = "", $kandang = "", $tahunBerdiri = "", $kaptenTim = "", $pelatih = "")
    {
        $this->nama = $nama;        
        $this->negara = $negara;        
        $this->kota = $kota;        
        $this->kandang = $kandang;        
        $this->tahunBerdiri = $tahunBerdiri;        
        $this->pelatih = $pelatih;        
        $this->kaptenTim = $kaptenTim;        
    }

    /* method set dan get */
        //method setter
    public function setNama($nama){
        $this->nama = $nama;
    }

    public function setNegara($negara){
        $this->negara = $negara;
    }

    public function setKota($kota){
        $this->kota = $kota;
    }

    public function setKandang($kandang){
        $this->kandang = $kandang;
    }

    public function setTahunBerdiri($tahunBerdiri){
        $this->tahunBerdiri = $tahunBerdiri;
    }

    public function setPelatih($pelatih){
        $this->pelatih = $pelatih;
    }

    public function setKaptenTim($kaptenTim){
        $this->kaptenTim = $kaptenTim;
    }

        //method getter
    public function getNama(){
        return $this->nama;
    }

    public function getNegara(){
        return $this->negara;
    }

    public function getKota(){
        return $this->kota;
    }

    public function getKandang(){
        return $this->kandang;
    }

    public function getTahunBerdiri(){
        return $this->tahunBerdiri;
    }

    public function getPelatih(){
        return $this->pelatih;
    }

    public function getKaptenTim(){
        return $this->kaptenTim;
    }

        //destructor
    public function __destruct()
    {
    }
}
?>

File: python/1st_week/Praktikum/latihanpraktikum1b_python/DataTim.py
class DataTim:
    # private atribut
    # nama = " ", negara = " ", kota, kandang, tahunBerdiri, pemain
    # nomorPunggung = 0
    def __init__(self, paramNama = "", paramNegara = "", paramKota = "", paramKandang = "", paramTahunBerdiri = "", paramPelatih = "", paramKapten = ""):
        self.nama = paramNama
        self.negara = paramNegara
        self.kota = paramKota
        self.kandang = paramKandang
        self.tahunBerdiri = paramTahunBerdiri
        self.pelatih = paramPelatih
        self.kapten = paramKapten
    
    # method set dan get
        # set method
    def setNama(self, paramNama):
        self.nama = paramNama
    
    def setNegara(self, paramNegara):
        self.negara = paramNegara

    def setKota(self, paramKota):
        self.kota = paramKota

    def setKandang(self, paramKandang):
        self.kandang = paramKandang

    def setTahunBerdiri(self, paramTahunBerdiri):
        self.tahunBerdiri = paramTahunBerdiri

    def setPelatih(self, paramPelatih):
        self.pelatih = paramPelatih

    def setKapteb(self, paramKapten):
        self.kapten = paramKapten

        # get method
    def getNama(self):
        return self.nama

    def getKandang(self):
        return self.negara

    def getNegara(self):
        return self.kota

    def getKota(self):
        return self.kandang
    
    def getTahunBerdiri(self):
        return self.tahunBerdiri
    
    def getPelatih(self):
        return self.pelatih
    
    def getKapten(self):
        return self.kapten

    def printData(self):
        print(f"Nama Tim : {self.nama}")
        print(f"Negara Asal Tim : {self.negara}")
        print(f"Kota Asal Tim : {self.kota}")
        print(f"Kandang Tim : {self.kandang}")
        print(f"Tahun Berdiri Tim : {self.tahunBerdiri}")
        print(f"Pelatih Tim : {self.pelatih}")
        print(f"Kapten Tim : {self.kapten}")
    
